ReleaseProcessAutomation/Git/CommandLineGitClient.cs

```diff
@@ -245,9 +245,14 @@ public string CheckoutNewBranch (string branchName)
     return checkout.Output;
   }
 
-  public void MergeBranch (string branchName, bool noCommit = false)
+  public void MergeBranch (string branchName, bool noCommit = false, bool fixConflicts = false)
   {
     var shouldCommit = noCommit ? "--no-commit" : "";
+    if (fixConflicts)
+    {
+      MergeBranchWithTheirsStrategy(branchName, noCommit);
+      return;
+    }
     var merge = ExecuteGitCommandWithOutput($"merge {branchName} --no-ff {shouldCommit}");
     if (!merge.Success)
     {
```

Comment:
Code structure:
```
if (fixConflicts)
{
}
else
{
  var shouldCOmmit = ...;
  var merge = ...;
  if (!merge.Success...)
  {
  ...
  }
}
```

Reasoning: those two paths are equal. 

Actually, given the flag, we could consider splitting those into separate methods since the boolean flag seems to split things down the middle. Only makes sense if a) the public behavior is diffferent enough so good naming is possible (indicates single repsonsibility) and b) shared functionality in the future is unlikely. 

Where do we use the noCommit flag? 

Where do we use fixConflicts = false?